web/client/utils/__tests__/MapUtils-test.js

```diff
@@ -1931,4 +1932,202 @@ describe('Test the MapUtils', () => {
+            "map": {
+                "mapOptions": {},
+                "layers": [
+                    {
+                        "id": "layer001",
+                        "thumbURL": "THUMB_URL",
+                        "search": {
+
+                        },
+                        "name": "layer001",
+                        "title": "layer001",
+                        "type": "wms",
+                        "url": "",
+                        "bbox": {
+
+                        },
+                        "visibility": true,
+                        "singleTile": false,
+                        "allowedSRS": {
+
+                        },
+                        "dimensions": [
+
+                        ],
+                        "hideLoading": false,
+                        "handleClickOnLayer": false,
+                        "catalogURL": "url",
+                        "useForElevation": false,
+                        "hidden": false,
+                        "params": {
+
+                        }
+                    }
+                ],
+                "groups": [],
+                "backgrounds": [
+                    {
+                        "id": "layer005",
+                        "thumbnail": "data"
+                    }
+                ]
+            },
+            "catalogServices": {},
+            "widgetsConfig": {},
+            "mapInfoConfiguration": {}
+        };
+        const map2 = {
+            "version": 2,
+            "map": {
+                "mapOptions": {},
+                "layers": [
+                    {
+                        "id": "layer001",
+                        "thumbURL": "THUMB_URL",
+                        "search": {
+
+                        },
+                        "name": "layer001",
+                        "title": "layer001",
+                        "type": "wms",
+                        "url": "",
+                        "bbox": {
+
+                        },
+                        "visibility": true,
+                        "singleTile": false,
+                        "allowedSRS": {
+
+                        },
+                        "dimensions": [
+
+                        ],
+                        "hideLoading": false,
+                        "handleClickOnLayer": false,
+                        "catalogURL": "url",
+                        "useForElevation": false,
+                        "hidden": false,
+                        "params": {
+
+                        }
+                    }
+                ],
+                "groups": [],
+                "backgrounds": [
+                    {
+                        "id": "layer005",
+                        "thumbnail": "data"
+                    }
+                ]
+            },
+            "catalogServices": {},
+            "widgetsConfig": {},
+            "mapInfoConfiguration": {}
+        };
+        expect(compareMapChanges(map1, map2)).toBeFalsy();
```

Comment:
Hi, this should be true instead of false. (the title of the test is suggesting this)